config/purposes/scrna_core_cdna_prep.yml

```diff
@@ -120,7 +120,11 @@ LRC GEM-X 5p cDNA PCR XP:
   # close off the submission when the plate is created
   :state_changer_class: StateChangers::AutomaticPlateStateChanger
   :work_completion_request_type:
-    ['limber_scrna_core_cdna_prep_gem_x_5p_gems_input', 'limber_scrna_core_cdna_prep_gem_x_5p']
+    [
+      'limber_scrna_core_cdna_prep_gem_x_5p_gems_input',
+      'limber_scrna_core_cdna_prep_gem_x_5p',
+      'limber_scrna_core_cdna_prep_gem_x_5p_gems_input_cite',
+    ]
```

Comment:
Don't think the above change is needed as we are not creating a new request type any more, we're using the existing one.